### ISSUE TITLE
Add Support for Wagtail 6

### ISSUE THREAD

**Issue by ephes** [ephes/django-cast#121]
- The `wagtailadmin/shared/field_as_li.html` template has been deprecated
- The `visuallyhidden` class does not work anymore (probably better for a11y)
